packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -564,9 +564,10 @@ describe('KeyringController', () => {
       { data: JSON.stringify(msgParams), from: account },
       SignTypedDataVersion.V4,
     );
-    const recovered = recoverTypedSignature_v4({
+    const recovered = recoverTypedSignature({
       data: msgParams as any,
```

Comment:
Using `primaryType: 'Mail' as const` should also remove the need for this as well.

Reply:
done [here](https://github.com/MetaMask/controllers/pull/914/commits/72149b509d1e174c837ce9e227568b9a52582a5f)